### DISCUSSION TITLE
Publishing to a private GitHub repo: fine-grained vs classic PAT

### DISCUSSION THREAD

**Discussion by pavelanni** [goreleaser/goreleaser#4845] [Ideas, Issue Triage, and General discussions]
It's not a question; I just wanted to share my experience, hoping it might help somebody. After several trial-and-error, I discovered that the new fine-grained personal access token didn't work for me, while the classic PAT worked just fine. In my case, I use my company's private repo, but the fine-grained PAT gives me access only to _my_ repos, and that was the reason I couldn't access it, as far as I understand.

This is the error I got:

`error=scm releases: failed to publish artifacts: could not release: POST https://api.github.com/repos/<company>/<project>/releases: 404 Not Found []`

The bottom line is to use Classic PAT with GoReleaser on GitHub.

**Comment by caarlos0** [comment]
Thanks for sharing, @pavelanni! ❤️

**Comment by tonymet** [comment]
has anyone gotten fine-grained working? I tried making one with content rw + PULL rw permissions and it failed . target publishing = winget (microsoft/winget-pkgs)

**Comment by vedantmgoyal9** [comment]
* https://github.com/vedantmgoyal9/winget-releaser/issues/172
